engine/storage_mongo_stordb.go

```diff
@@ -1572,10 +1572,13 @@ func (ms *MongoStorage) RemoveVersions(vrs Versions) (err error) {
 	if len(vrs) == 0 {
 		return ms.query(func(sctx mongo.SessionContext) (err error) {
 			dr, err := ms.getCol(ColVer).DeleteOne(sctx, bson.D{})
```

Comment:
if you are returning default error here, out of good practice please don't use := but = when capturing err